rust/worker/src/blockstore/arrow/blockfile.rs

```diff
@@ -242,6 +242,23 @@ impl<'me, K: ArrowReadableKey<'me>, V: ArrowReadableValue<'me>> ArrowBlockfileRe
         }
     }
 
+    // Count the total number of records.
+    pub(crate) async fn count(&self) -> Result<usize, Box<dyn ChromaError>> {
```

Comment:
can we add a basic sanity test here

Reply:
Was in the process of writing when you gave this comment. Uploaded